File: 240524/숫자들의 배수/multiple-of-numbers.py
n = int(input())
arr = []

i = 1
while True:
    cnt = 0
    for _ in arr:
        if _%5 == 0:
            cnt += 1
    if cnt == 2:
        break
    arr.append(n*i)
    i += 1

for j in arr:
    print(j, end=' ')

File: 241224/숫자들의 배수/multiple-of-numbers.cpp
#include <iostream>
using namespace std;

int main() {
    // 여기에 코드를 작성해주세요.
    int num, arr[100], cnt=0;
    cin >> num;

    for (int i=0; i<10; i++) {
        arr[i] = num*(i+1);
        cout << arr[i] << ' ';
        if (arr[i]%5 == 0) {
            cnt++;
        }
        if (cnt == 2) {
            break;
        }
    }
    return 0;
}